src/pcss-reports-api/src/main/resources/wsdl/pcssReport.wsdl.renamed

```diff
@@ -0,0 +1,188 @@
+                </xsd:restriction>
+            </xsd:simpleType>
+        </xsd:schema>
+    </wsdl:types>
+    <wsdl:message name="pcssReport_PortType_getJustinReportResponse">
+        <wsdl:part name="parameters" element="tns:getJustinReportResponse">
+    </wsdl:part>
+    </wsdl:message>
+    <wsdl:message name="pcssReport_PortType_getJustinAdobeReportResponse">
+        <wsdl:part name="parameters" element="tns:getJustinAdobeReportResponse">
+    </wsdl:part>
+    </wsdl:message>
+    <wsdl:message name="pcssReport_PortType_getJustinAdobeReport">
+        <wsdl:part name="parameters" element="tns:getJustinAdobeReport">
+    </wsdl:part>
+    </wsdl:message>
+    <wsdl:message name="pcssReport_PortType_getJustinReport">
+        <wsdl:part name="parameters" element="tns:getJustinReport">
+    </wsdl:part>
+    </wsdl:message>
+    <wsdl:portType name="pcssReport_PortType">
+        <wsdl:operation name="getJustinReport">
+            <wsdl:input message="tns:pcssReport_PortType_getJustinReport">
+    </wsdl:input>
+            <wsdl:output message="tns:pcssReport_PortType_getJustinReportResponse">
+    </wsdl:output>
+        </wsdl:operation>
+        <wsdl:operation name="getJustinAdobeReport">
+            <wsdl:input message="tns:pcssReport_PortType_getJustinAdobeReport">
+    </wsdl:input>
+            <wsdl:output message="tns:pcssReport_PortType_getJustinAdobeReportResponse">
+    </wsdl:output>
+        </wsdl:operation>
+    </wsdl:portType>
+    <wsdl:binding name="JusticePCSSCommon_wsProvider_pcssReport_Binder" type="tns:pcssReport_PortType">
+        <soap12:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        <wsdl:operation name="getJustinReport">
+            <soap12:operation soapAction="JusticePCSSCommon_wsProvider_pcssReport_Binder_getJustinReport" style="document"/>
+            <wsdl:input>
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:input>
+            <wsdl:output>
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:output>
+        </wsdl:operation>
+        <wsdl:operation name="getJustinAdobeReport">
+            <soap12:operation soapAction="JusticePCSSCommon_wsProvider_pcssReport_Binder_getJustinAdobeReport" style="document"/>
+            <wsdl:input>
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:input>
+            <wsdl:output>
+                <soap12:body parts="parameters" use="literal"/>
+            </wsdl:output>
+        </wsdl:operation>
+    </wsdl:binding>
+    <wsdl:service name="pcssReport">
+        <wsdl:port name="JusticePCSSCommon_wsProvider_pcssReport_Port" binding="tns:JusticePCSSCommon_wsProvider_pcssReport_Binder">
+            <soap12:address location="http://pcssreports.pcss.open.gov.bc.ca:5555/ws/JusticePCSSCommon.wsProvider:pcssReport/JusticePCSSCommon_wsProvider_pcssReport_Port"/>
+        </wsdl:port>
+    </wsdl:service>
+</wsdl:definitions>
```

Comment:
missing new line

Reply:
also know that when I point these out its more my own OCD/PTSD. If the files are auto generated, don't worry.

Reply:
those are coming from webMethods directly

Reply:
Got rid of that file entirely, as I had left it in by mistake anyhow.